proto/spire/next/api/entry/v1/entry.proto

```diff
@@ -144,16 +144,13 @@ message UpdateEntryRequest {
 }
 
 message BatchUpdateEntryRequest {
-    // The entries to be updated. The entry ID field is output only, and will
-    // be ignored here.
+    // The entries to be updated.
```

Comment:
How do we communicate input-only/output-only fields on fields?

Reply:
If the field of the field is categorically output or input only, then the comment describing the field of the field can have "Output only", etc. If it depends on the RPC, then a think a comment on the field in the RPC is about as good as you can get.